client/src/components/AudioDisplayTable/AudioDisplayTable.jsx

```diff
@@ -125,40 +126,59 @@ class AudioDisplayTable extends Component {
         let newIsExpanded = {
             ...this.state.isExpanded,
             [player.playerName]: !this.state.isExpanded[player.playerName]
-        }
+        };
         this.setState({
             isExpanded: newIsExpanded
-        })
-    };
+        });
+    }
 
     render() {
         const { classes } = this.props;
-
+        let baselinePlayer = this.state.baselinePlayerSet ? this.props.game.baselinePlayer : null;
```

Comment:
isnt this.props.game.baselinePlayer null anyways if it wasnt set?

Reply:
probs